apps/safe-claiming-app/src/hooks/useDarkMode.ts

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useMemo, useState } from "react"
+import initTheme from "src/config/theme"
+
+const isSystemDarkMode = (): boolean => {
+  if (typeof window === "undefined" || !window.matchMedia) return false
+  return window.matchMedia("(prefers-color-scheme: dark)").matches
+}
+
+export const useDarkMode = (): boolean => {
+  const location = window.location
+
+  const [isDarkMode, setIsDarkMode] = useState<boolean>(false)
+
+  useEffect(() => {
+    setIsDarkMode(location.hash.endsWith("+dark") ?? isSystemDarkMode())
```

Comment:
For me this does not work because it always sets it to `false` if there is no +dark in the hash, even with system dark mode true. Either use `||`instead of `??` or this should rather be done via query parameter.
Alternatively we could only support SystemDarkMode until darkmode settings are passed through the safe-apps-sdk

Reply:
are you running https://github.com/safe-global/web-core/pull/1219 locally? I included the way to test it in the Description